EXPLAINER.md

```diff
@@ -93,14 +93,14 @@ info when users tap NFC devices can be done smoothly once permission is granted.
 
 ```js
 const reader = new NDEFReader();
```

Comment:
Use `const ndef` here and below.

Reply:
Hmm. Looks like these changes have been lost in the revert, need to re-apply.